cmd/sync/sync.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	cmdp "github.com/spacemeshos/go-spacemesh/cmd"
+	"github.com/spacemeshos/go-spacemesh/log"
+	"github.com/spacemeshos/go-spacemesh/mesh"
+	"github.com/spacemeshos/go-spacemesh/p2p"
+	"github.com/spacemeshos/go-spacemesh/sync"
+	"github.com/spacemeshos/go-spacemesh/timesync"
+	"github.com/spf13/cobra"
+	"os"
+	"time"
+)
+
+var lg = log.New("sync_test", "", "")
```

Comment:
it is better you do this after calling app.`initialize` because some log settings (e.g debug, json). are set only after initialization.